templates/generic_show_form.html.twig

```diff
@@ -541,6 +541,20 @@
                           {% elseif field is same as 'last_boot' and item.isField('is_dynamic') and item.fields['is_dynamic'] %}
                               {# display last_boot data only if item is dynamic and have field #}
                               {{ fields.htmlField('last_boot', item.fields['last_boot']|formatted_datetime(true), __('Last boot date')) }}
+                          {% elseif field is same as 'minimal_required_validation_percent' %}
```

Comment:
If this field is only for validations, it should probably not be in the generic form. The specific itemtype should have a separate template that extends the generic one.